testsuite/SystemInfo/btrfs-filesystem-show.cc

```diff
@@ -81,9 +81,9 @@ BOOST_AUTO_TEST_CASE(parse_good)
     };
 
     vector<string> output = {
-	"uuid:ea108250-d02c-41dd-b4d8-d4a707a5c649 devices:</dev/mapper/system-test>",
-	"uuid:d82229f2-f9e4-40fd-b15f-84e2d42e6d0d devices:</dev/mapper/system-testsuite>",
-	"uuid:653764e0-7ea2-4dbe-9fa1-866f3f7783c9 devices:</dev/mapper/system-btrfs>"
+	"uuid:ea108250-d02c-41dd-b4d8-d4a707a5c649 devices:<{ id:1 name:/dev/mapper/system-test }>",
+	"uuid:d82229f2-f9e4-40fd-b15f-84e2d42e6d0d devices:<{ id:1 name:/dev/mapper/system-testsuite }>",
+	"uuid:653764e0-7ea2-4dbe-9fa1-866f3f7783c9 devices:<{ id:1 name:/dev/mapper/system-btrfs }>"
```

Comment:
previous output was wrong? If it was valid then maybe it helps to test also other format without id as now I see only outputs with id in it.

Reply:
ah, nevermind. I see that it is output of storage and not cmd output. cmd output is called input which confuse me, but it is consistent so fine for me.

Reply:
This is the output after parsing to verify that the parsing work correctly. The "id" is added in the PR.